src/main/resources/standardTestCases/empty_string/empty_string.wdl

```diff
@@ -0,0 +1,33 @@
+task hello {
+  command {
+    echo "Hello!"
+  }
+  output {
+    String empty = ""
+  }
+  runtime {
+    docker: "ubuntu:latest"
+  }
+}
+
+task goodbye {
+  String emptyInputString
+  command {
+    echo "${emptyInputString}"
+  }
+  output {
+    String empty = read_string(stdout())
+  }
+  runtime {
+   docker: "ubuntu:latest"
+  }
+}
+
+workflow hello {
+  call hello
+  call goodbye {input: emptyInputString=hello.empty }
```

Comment:
I'm trying to not change the tests at the moment, except in places where I need to for one reason or another. But I agree that something like that would be good